python/dlisio/lis/curves.py

```diff
@@ -20,7 +20,7 @@
 }
 
 
-def curves(f, dfsr, strict=True):
+def curves(f, dfsr, strict=True, skip_fast=False):
```

Comment:
If feature is planned to be permanent, add some test case based on the most simple file?